docker/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM stanback/alpine-samba
+
+COPY ./smb.conf /etc/samba/smb.conf
+WORKDIR /srv/samba/shared
+RUN chmod 777 -R /srv/samba/shared
+RUN adduser -D -g '' wearefrank
+
```

Comment:
I would expect this docker setup in a subdirectory in the test-integration module.